com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -49,9 +49,11 @@ void FixedUpdate()
         "docs/Learning-Environment-Design.md")]
     public class Academy : IDisposable
     {
-        const string k_ApiVersion = "API-15-dev0";
+        // TODO move the versions to Constants.cs?
+        const string k_ApiVersion = "0.15.0";
+        internal const string k_PackageVersion = "0.14.1-preview";
```

Comment:
k_ApiVersion and k_PackageVersion are not the same here. What is the process behind updating those?

Reply:
yeah, maybe move to constants with comments or links to how to update each one.

Reply:
Added comments.